index.html

```diff
@@ -9,42 +9,39 @@
   </head>
   <body>
     <h1 class="title">Calculator</h1>
-    <div class="calc-background">
-      <section class="calc-row">
-        <p class="answer-number">0</p>
-      </section>
+    <article class="calculator">
+      <div class="calc-row">
+        <strong class="answer-number">0</strong>
```

Comment:
These two html elements can be merged into 1. It is also not necessary to use flex here.